testscripts/commands/gen.txt

```diff
@@ -3,7 +3,7 @@
 env NUM_COMMANDS=44
 
 hugo gen -h
-stdout 'Generate documentation and syntax highlighting styles\.'
+stdout 'Generate documentation for your project using Hugo''s documentation engine, including syntax highlighting for various programming languages\.'
```

Comment:
If the test passes then this is fine, but in general these text assertions are ... smoke tests, and testing shorter substrings is most often good enough -- which makes the test more stable when things changes.

Reply:
I checked it locally, they do pass the test, should I just asset against a portion of the string?

Reply:
maybe something like;
stdout 'Generate documentation for your project'